web/content/release-2.5.adoc

```diff
@@ -0,0 +1,26 @@
+:FOREMAN_VER: 2.5
```

Comment:
It would be great if this could actually be pulled from the 2.5 branch somehow. Let's work on that in a follow up.

Reply:
Yeah, let's just not waste too much time on improving Makefiles or this as the ultimate goal should be to use Antora. That is next-level documentation: https://docs.antora.org/antora/latest/